vnext/PropertySheets/External/Microsoft.ReactNative.Uwp.Common.props

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- 
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  Licensed under the MIT License.
+
+  This file will be consumed by ALL UWP app and module projects (both inside
+  and outside of this repo) that build on top of Microsoft.ReactNative.
+  Do not make any changes here unless it applies to ALL such projects.
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildThisFileDirectory)Microsoft.ReactNative.Common.props" />
```

Comment:
see for ex here, we don't use a trailing backslash after MSBuildThisFileDirectory

Reply:
That is a essentially a read-only value that is always set correctly.